docs/reference/contracts/apis/managed.md

```diff
@@ -0,0 +1,294 @@
+
+## Circuit Breakers
+
+### `getCircuitBreakerState`
+
+```solidity
+getCircuitBreakerState(IERC20 token)
+returns (uint256 bptPrice,
+         uint256 referenceWeight,
+         uint256 lowerBound,
+         uint256 upperBound,
+         uint256 lowerBptPriceBound,
+         uint256 upperBptPriceBound)
+```
+
+Returns the full circuit breaker state for a given `token`. Implemented in `ManagedPoolSettings`.
+
+### `setCircuitBreakers`
+
+```solidity
+setCircuitBreakers(IERC20[] memory tokens,
+                   uint256[] memory bptPrices,
+                   uint256[] memory lowerBoundPercentages,
+                   uint256[] memory upperBoundPercentages)
+
+emit CircuitBreakerSet(IERC20 indexed token,
+                       uint256 bptPrice,
+                       uint256 lowerBoundPercentage,
+                       uint256 upperBoundPercentage)
+```
+
+Sets a current circuit breaker for one or more `tokens`. The lower and upper bound percentages correspond to a relative change in the token's spot price. An example of this would be that a lower bound of 0.8 means the breaker should prevent trades that result in the value of the token dropping 20% or more relative to the rest of the pool. This is a permissioned function that can only be called by an authorized address set by the `owner`. Implemented in `ManagedPoolSettings`.
+
+## Add/Remove Tokens
+
+### Add Token
+
+```solidity
+addToken(IERC20 tokenToAdd,
+         address assetManager,
+         uint256 tokenToAddNormalizedWeight,
+         uint256 mintAmount,
+         address recipient)
+
+emits TokenAdded(IERC20 indexed token, uint256 normalizedWeight)
+```
+
+Adds a token, `tokenToAdd`, to the pool's list of tradeable tokens. This is a permissioned function that can only be called by an authorized address set by the `owner`. Implemented in `ManagedPoolSettings`.
+
+::: warning
+When adding a new token to a pool, the weights of all other tokens will be decreased. The new token will have a balance of zero, so it is up to the owner to provide some imediate liquidity after calling this function.
+:::
+
+::: info 
+**Note:** Regular joins do not work when a new token has a balance of zero. The only way to deposit an initial amount is by using an Asset Manager.
+:::
+
+
+### Remove Token
+
+```solidity
+removeToken(IERC20 tokenToRemove,
+            uint256 burnAmount,
+            address sender)
+
+emits TokenRemoved(IERC20 indexed token)
+```
+
+Removes a token, `tokenToRemove`, to the pool's list of tradeable tokens. This is a permissioned function that can only be called by an authorized address set by the `owner`. Implemented in `ManagedPoolSettings`.
+
+::: info
+**Note:** Tokens can only be removed if the pool has more than two tokens (not including BPT).
```

Comment:
like with add token, add note about how removing token increases all other tokens' weights and any remaining token balance must be removed via asset manager